docs/06-concepts/02-models.md

```diff
@@ -243,18 +265,21 @@ fields:
   dateTimeDefault: DateTime, default=now
 
   ### Sets the default value for a boolean field.
-  boolDefault: bool, defaultModel=false, defaultDatabase=true
+  boolDefault: bool, defaultModel=false, defaultPersist=true
 
   ### Sets the default value for an integer field.
-  intDefault: int, defaultDatabase=20
+  intDefault: int, defaultPersist=20
 
   ### Sets the default value for a double field.
-  doubleDefault: double, default=10.5, defaultDatabase=20.5
+  doubleDefault: double, default=10.5, defaultPersist=20.5
 
   ### Sets the default value for a string field.
   stringDefault: String, default="This is a string", defaultModel="This is a string"
+
+  ### Sets the default value for a UUID field.
+  uuidDefault: UuidValue, default=random, defaultModel=random, defaultPersist=random
```

Comment:
Question: Does this add any value to the point proven in this section? Otherwise it could probably be removed to keep the example as simple as possible.